flavors/standard-EXASOL-all/flavor_base/build_steps.py

```diff
@@ -177,6 +177,9 @@ def requires_tasks(self):
     def get_path_in_flavor(self):
         return "flavor_base"
 
+    def get_language_definition(self) -> str:
+        return "language_definitions.json"
+
```

Comment:
```suggestion
    def get_language_definition(self) -> str:
        return "language_definitions.json"
```

Reply:
Hm, what is the change here?

Reply:
3 empty lines between the function and the following class.

Reply:
Done